PocketKit/Sources/PocketKit/PocketAppDelegate.swift

```diff
@@ -79,4 +81,14 @@ public class PocketAppDelegate: UIResponder, UIApplicationDelegate {
 
         return true
     }
+
+    /// Sets orientations to use for the views
+    /// - Parameters:
+    ///   - application: singleton app object
+    ///   - window: window whose interface orientations you want to retrieve
+    /// - Returns: orientations to use for the view
+    public func application(_ application: UIApplication, supportedInterfaceOrientationsFor window: UIWindow?) -> UIInterfaceOrientationMask {
```

Comment:
@cyndichin does this get called every time there's a view controller change (i.e appear, disappear)?

Reply:
@dskuza yes it does but it also provides the simplest way to set all views to portrait from what I can find, do you have an alternative way that might be preferred?

Reply:
@cyndichin not that I know of in a SwiftUI world; the "old" setup was based on UIKit delegate methods

Reply:
That, and all of the usage of  `UITabBarControllerDelegate` and `UINavigationControllerDelegate`.